app/herdbook.py

```diff
@@ -7,12 +7,28 @@
 """
 
 import sys
+import uuid
 import time
 from flask import Flask, render_template, jsonify
 
 import utils.database as db
 
 APP = Flask(__name__, static_folder="/static")
+APP.secret_key = uuid.uuid4().hex
```

Comment:
it's used for signing the session cookie. You often set a static secret (since it allows the same cookies to be used for multiple servers, etc.), but I figured a random one would to for now... It should be moved to the settings file later though.